tests/wpt/mozilla/tests/mozilla/window_performance_topLevelDomComplete.html

```diff
@@ -7,11 +7,17 @@
 <body>
 <script>
 async_test(function(t) {
-  window.onload = t.step_func(function() {
-    assert_true(performance.timing.domLoading <= performance.timing.topLevelDomComplete);
-    assert_true(performance.timing.topLevelDomComplete <= performance.timing.domComplete);
-    t.done();
-  });
+  var observer = new PerformanceObserver(
```

Comment:
PerformanceObserver takes a callback argument, but this is not providing one. Did you run this test locally with these changes?

Reply:
Also this test never calls observer.observe.

Reply:
Actually, I wonder if `window.onload = t.step_func(..` evaluates to a callback argument, making this actually work. In any case, that doesn't look intentional to me.

Reply:
This is relevant again; we should be passing `t.step_func(function() {` as the argument to PerformanceObserver.